### DISCUSSION TITLE
Multi-year renewal for commonhaus.dev

### DISCUSSION THREAD

**Discussion by ebullient** [commonhaus/foundation#182] [Council Meeting Topics]
.dev domain costs will go up next year. 

- Should we renew now?
- If so, for how many years?

**Comment by sigrunixia** [comment]
Do you have the amount of the expected increase for the domain?

**Reply by evanchooly** [reply]
it's going up by less than $2.

**Reply by ebullient** [reply]
(too many windows)

Price change is in August. Just something to talk about, and do if we decide to. ;)

**Comment by kenfinnigan** [comment]
I would say it doesn't hurt to get a couple of extra years while it's a bit cheaper, but it's not going up a crazy amount either

**Comment by ebullient** [comment]
I realized it was also up for renewal anyway (I'd only grabbed a year when this was just an idea).

Job done, will submit expenses.
